pages/mbs.html

```diff
@@ -0,0 +1,172 @@
+              </p>
+
+              <h4>Setting up</h4>
+              <p>Mast was able to get MBS Lending up and running quickly. <strong>“The onboarding process was slick, efficient and Mast were incredibly observant and attentive to our needs and requirements.”</strong> Despite the specialist products, policies, and bespoke forms,  Mast had the MBS Lending pilot portal up and running within 12 weeks. This also included custom work on affordability calculations and user management. “It was a pleasure to work with a provider that will go the extra mile to say yes.”
+              </p>
+
+
+              <p>Once live, the impact was obvious. “Mast helped us to <strong>increase our application to completion conversion from 51% in 2021 when the process was paper based, to 73% in 2022</strong>. This has proved to be hugely valuable and has improved our internal efficiencies considerably.” Mast’s live policy prompts and bespoke packaging requirements helped remove more speculative applications earlier on and MBS Lending “are now able to focus on providing quality service and underwriting applications where we are much more likely to lend.”
+              </p>
+
+              <h4>Life with Mast</h4>
+
+              <blockquote class="d-none d-lg-block bg-primary-alt" style="float: right; margin: -4rem -7rem 1.25rem 1.25rem; width: 60%">
+                <div class="h4 mb-2">
+                  &ldquo;It has been great to work in true partnership with Mast who really took the time to understand our business. We have been very impressed with Mast's responsiveness to our requirements and their passion to support our change agenda.&rdquo;
+                </div>
+                <span class="text-small text-muted">Simon Taylor, CEO, Melton Building Society</span>
+              </blockquote>
+
+              <p>
+                Collaboration with lenders doesn’t end on go-live day with Mast, it’s just the start. “<strong>The platform continues to go from strength to strength</strong>. Whether that be with improvements to both the broker and lender user journey, third party integration, sales data, product management, lending rules.” With regular check-ins and user research calls, “Mast never rest on their laurels and are continuously working hard to improve every part of the system and to build on what is already an efficient, modern and intuitive platform.”
+              </p>
+
+              <p>
+                Mast have delivered regular updates to functionality, application forms, affordability calculators, user permissions and UX designs and will continue to do so for all clients past, present and future.
```

Comment:
I don't know if its me but something about this line here doesn't feel right. I think it's because it's the last line/conclusion of the case study but is a little bland on its own. It should be probably be mixed in somehow with the previous paragraph, and cutting some of it down to avoid it being too long.

Reply:
Also presumably we aren't going to be delivering updates for past clients...